src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestGeneration.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Diagnostics.Tracing;
+using Xunit;
+using System.Reflection;
+
+//using Mdt = MdtEventSources;
+using Sdt = SdtEventSources;
+using System.Diagnostics;
+using System.Threading;
+using System.Text.RegularExpressions;
+
+namespace BasicEventSourceTests
+{
+    
+    public class TestsManifestGeneration
+    {
+        /// <summary>
+        /// Visual Studio Online bug #125529, EventSource would fail when an EventSource was named "EventSource".
+        /// </summary>
+        [Fact]
+        public void Test_EventSource_NamedEventSource()
+        {
+            lock (TestUtilities.EventSourceTestLock)
+            {
+                using (var es = new SdtEventSources.DontPollute.EventSource())
+                {
+                    using (var el = new LoudListener())
+                    {
+                        int i = 12;
+                        es.EventWrite(i);
+
+                        Assert.Equal(1, LoudListener.LastEvent.EventId);
+                        Assert.Equal(1, LoudListener.LastEvent.Payload.Count);
+                        Assert.Equal(i, LoudListener.LastEvent.Payload[0]);
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Validate generating manifests for the inproc as well as reflection-loaded event sources
+        /// </summary>
+        [Fact]
+        public void Test_GenerateManifest()
+        {
+#if true
+            Console.WriteLine("Disabled because it fails in checkin but not locally on 10/6/2015");
```

Comment:
Tests should be disabled with an attribute rather than compilation constants, e.g.
```C#
[ActiveIssue(12345)] // where 12345 is the GitHub issue number
```